exchange/exchange_test.go

```diff
@@ -1400,7 +1400,7 @@ func TestBidReturnsCreative(t *testing.T) {
 
 	//Run tests
 	for _, test := range testCases {
-		resultingBids, resultingErrs := e.makeBid(sampleBids, sampleAuction, test.inReturnCreative)
```

Comment:
Added new unit test for this: TestMakeBidsVideoAttributes

Reply:
I am not sure if we need `TestMakeBidsVideoAttributes` because all those test cases are already being tested in the `TestMakeBidExtJSON` test that you added. In `makeBids` we only pass `impExtInfo` downstream, we don't really have any logic addition/deletion/modification there. In such cases I'd just add one new test case for that upstream method (in this case `makeBids`) to make sure the value is being passed downstream as expected. Exhaustive test cases about the downstream method (in this case `makeBidExtJSON`) where the actual logic is changed makes sense only for the test for that method. This is why I recommended that we just add a new test case for `buildBidResponse` or modify and existing test case for `buildBidResponse` to just make sure that the value is properly passed all the way downstream to `makeBidExtJSON`. 

I hope this makes sense!